src/api/operations/form.queries.graphql

```diff
@@ -75,6 +75,53 @@ query GetFormDefinitions(
   }
 }
 
+query GetFormIdentifierForEditor($identifier: String!) {
+  formIdentifier(identifier: $identifier) {
+    id
+    identifier
+    displayVersion {
+      ...FormDefinitionFieldsForEditor
+    }
+    published {
+      ...FormDefinitionFieldsForEditor
+    }
+    draft {
+      ...FormDefinitionFieldsForEditor
+    }
+    allVersions {
```

Comment:
I don't see this being used at all, can we remove it?
Presumably the version history table will use it's own query for a paginated table, using GenericTableWithData to perform the query and pagination logic. I don't see that the version history needs to be part of this query.

Reply:
Removed!